### ISSUE TITLE
Importing MIDI?

### ISSUE THREAD

**Issue by GabrielMejiaEstrella** [leafo/sightreading.training#109]
Hello! 

I absolutely love sight reading trainer, but I notice that the only way to create a new play along-song is by inputting each note with LML. I don't know anything about coding, so I don't know how feasible this would be, but one feature I'd like to see is being able to import midi data and having it automatically create a play-along. It seems like that would be a lot more practical than typing in one note at a time.

**Comment by RayReddingtonx** [comment]
absolutely. I tried to learn sight reading using Musescore but it doesn't has any tutor feature. Musescore however has tones of xml etudes for sight reading. It will be great if we can import xlm or midi or musescore files or if this tool simply copy the basic etudes all into its system and categorize them with levels of games. So, we can gradually up the level and improve with it. No need to import midi when all the basic etudes can be found inbuilt its generator. Also, the rhythm/time accent improvement is needed plus metronome. 
https://musescore.com/user/58480/scores/1920636

**Comment by jamesdeluk** [comment]
I came here to post this too! Definite +1.
